x/proof/keeper/msg_server_submit_proof.go

```diff
@@ -1,22 +1,56 @@
 package keeper
 
 import (
+	"bytes"
 	"context"
+	"crypto/sha256"
 
+	"github.com/pokt-network/smt"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"github.com/pokt-network/poktroll/pkg/relayer/protocol"
 	"github.com/pokt-network/poktroll/x/proof/types"
+	servicetypes "github.com/pokt-network/poktroll/x/service/types"
+	sessionkeeper "github.com/pokt-network/poktroll/x/session/keeper"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
 )
 
+const (
+	// relayMinDifficultyBits is the minimum difficulty that a relay must have
+	// to be reward (i.e. volume) applicable.
+	// TODO_BLOCKER: relayMinDifficultyBits should be a governance-based parameter
+	relayMinDifficultyBits = 0
+
+	// sumSize is the size of the sum of the relay request and response in bytes.
+	// This is needed to extract the relay request and response from the closest
+	// merkle proof.
+	// TODO_TECHDEBT(@h5law): Add a method on the smst to extract the value hash
```

Comment:
Addressed in https://github.com/pokt-network/smt/pull/43 you can update to use this hash of the smt repo in go mod for all the functions e3dbbbd9f97d4f9007c67f5c7e512a93c8f3335c